tests/resources/rebuild_example_data

```diff
@@ -112,8 +91,6 @@ def main(ctx, rebuild_input):
         # The environ here should be the same (modulo the tempdir location)
         # as we use in test_examples.py.
         environ = {'ZIPLINE_ROOT': d.getpath('example_data/root')}
-        if rebuild_input:
-            rebuild_input_data(environ)
```

Comment:
We'll need to switch it to use a different bundle, so for now if someone passes the `--rebuild-input` flag we raise a `StandardError`and let users know that this will be fixed in a future release